tests/unit/matrix/test_subtotals.py

```diff
@@ -519,17 +519,27 @@ def _table_margin_prop_(self, request):
 class DescribeZscoreSubtotals(object):
     """Unit test suite for `cr.cube.matrix.ZscoreSubtotals` object."""
 
-    def it_provides_access_to_the_base_counts_to_help(self, cube_result_matrix_):
-        cube_result_matrix_.weighted_counts = [[1, 2], [3, 4]]
-        subtotals = ZscoreSubtotals(None, None, cube_result_matrix_)
-        assert subtotals._base_counts == [[1, 2], [3, 4]]
+    def it_provides_an_intersections_interface_method(
+        self, request, _init_, _base_weighted_counts_
+    ):
+        _base_weighted_counts_.weighted_counts = [[1, 2], [2, 3]]
+        _base_weighted_counts_.rows_margin = None
+        _base_weighted_counts_.columns_margin = [3, 4]
+        _base_weighted_counts_.table_margin = 12
+        property_mock(
+            request,
+            ZscoreSubtotals,
+            "_intersections",
+            return_value=np.array([[1, 2], [3, 4]]),
+        )
+
+        intersections = ZscoreSubtotals.intersections(_base_weighted_counts_, None)
+
+        _init_.assert_called_once_with(ANY, _base_weighted_counts_, None, True, True)
+        assert intersections.tolist() == [[1, 2], [3, 4]]
```

Comment:
Why does `ZscoreSubtotals` need a `.intersections()` interface method? I'm not finding any callers.

It does however have a `.blocks()` interface method and I don't see a test for that here, maybe just convert this one into that and remove the intersections classmethod.